README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_task_generate_table_DOM/)
+    - [DEMO LINK](https://SerhiiSemennikov.github.io/js_task_generate_table_DOM/)
```

Comment:
Please, provide a link to the DEMO in the PR description and make sure that it works